lib/axios.types.ts

```diff
@@ -0,0 +1,7 @@
+export {
+  AxiosInstance,
+  AxiosPromise,
+  AxiosRequestConfig,
+  AxiosResponse,
+  CancelTokenSource,
```

Comment:
```suggestion
  CancelTokenSource,
  AxiosResponse,
  AxiosError,
```

Reply:
@dammy001 Thanks for the suggestion!
The suggested change adds `AxiosResponse` again which it's a duplicate. please edit that, then we can commit🙏

Reply:
Yeah. Updated now @mohsenbostan